src/PostSharp.LicenseServer/Admin/AddLicense.aspx.cs

```diff
@@ -28,7 +29,18 @@ protected void AddButton_OnClick( object sender, EventArgs e )
                 return;
             }
 
-            if ( !parsedLicense.IsLicenseServerElligible() )
+            if ( parsedLicense.MinPostSharpVersion > ApplicationInfo.Version )
```

Comment:
I don't think this is useful, I actually think it is harmful. As long as the server can parse the key, this is okay,